system/jlib/jptree.hpp

```diff
@@ -239,7 +242,7 @@ jlib_decl void addPTreeItem(IPropertyTree *ptree, const char * name, const char
 
 jlib_decl IPropertyTree *createPTree(MemoryBuffer &src, byte flags=ipt_none);
 jlib_decl IPropertyTree *createPTreeFromBinary(IBufferedSerialInputStream &src, byte flags=ipt_none);
-jlib_decl IPropertyTree *createPTreeFromBinary(IBufferedSerialInputStream &src, IPTreeNodeCreator *nodeCreator=nullptr);
+jlib_decl IPropertyTree *createPTreeFromBinary(IBufferedSerialInputStream &src, IPTreeNodeCreator *nodeCreator);
```

Comment:
One of the functions needs to have a non-default second parameter, otherwise neither can be called when only one argument is supplied.

Reply:
I'd be tempted to take both default args off at this point.